src/aurelia-loader-webpack.ts

```diff
@@ -127,7 +84,6 @@ export class WebpackLoader extends Loader {
       const registry = __webpack_require__.c;
       const cachedModuleIds = Object.getOwnPropertyNames(registry);
       cachedModuleIds
-        .filter(moduleId => typeof moduleId === 'string')
```

Comment:
As I wrote in the comment, `Object.getOwnPropertyNames` always returns an array of string. This filter will never remove an item.